applications/RomApplication/tests/test_compressible_potiential_rom.py

```diff
@@ -1,5 +1,5 @@
 import os
-
+import numpy as np
```

Comment:
Why is this required if not used in this file?

Reply:
For that particular test it was required but not being imported. The failing test did not trigger any alarm because the RomApplication does not apper in the Continuous Integration any longer

Reply:
OK, but didn't we add the `RomApplication` to the CI at some point?

Reply:
We just spoke with @roigcarlo and found out that the RomApplication is indeed still in the CI. It just so happens that the "nightly" test suite is empty. We only have the tests in the "small" one. We shall add it in a future breve PR